### PR TITLE
implemented improvements for windows requirements tool

### PR DESCRIPTION
**Pull request by d-casal**
Addressed [issue7](https://github.com/green-code-initiative/ecoCode-common/issues/7)

### CODE REVIEW

File: tools/check_requirements/README.md [hunk 1]
```diff
@@ -4,17 +4,17 @@
 - [Requirements](#requirements)
 - [Files](#files)
 - [How does it work ?](#how-does-it-work-)
-- [Screenshot](#screenshot)
+- [Screenshot](#screenshots)
```
Comment (dedece35) [inline tools/check_requirements/README.md:7]: "Screenshots" instead of "Screenshot"

**Review by dedece35 — CHANGES_REQUESTED** [review]

**Review by dedece35 — CHANGES_REQUESTED** [review]
here is my test screenshot on Windows 11 Family
maybe some characters to correct
Windows 11 family 23H2

File: tools/check_requirements/README.md (hunk 1 above)
Comment (d-casal) [reply tools/check_requirements/README.md]: Updated as requested

**Comment by d-casal** [comment]
Hi David, can you double check that you are running my script (and not a different working copy)? I do not have ***** or special characters in my script and I use a different wording for messages. It looks like perhaps you tested a different version, not the one from my pull request?

image

image

Let me know :-)

**Comment by dedece35** [comment]
sorry .... my bad ... I didn't check on your PR branch. sorry.
quite good now.
but can we check GIT installation even if "javac" is'nt OK, in order to show all fails with one execution ?

Windows 11 family 23H2 - BIS

**Comment by d-casal** [comment]
yes, good point. I updated the Pull Request. Should behave as described in your previous comment now:

image

**Comment by dedece35** [comment]
Hi @d-casal,
thank you for your work !
now it's ok for me.
Windows 11 family 23H2 - new

**Comment by d-casal** [comment]
@dedece35 implemented those suggestions for MacOS 👍

**Review by dedece35 — APPROVED** [review]

File: tools/check_requirements/README.md [hunk 2]
```diff
@@ -26,23 +26,36 @@ Tested on :
 
 For Mac OSX and Linux :
 
-- `_config.sh` : file containing the configuration of checker tool (tools, versions, debug)
+- `_config.sh` : file for loading configuration parameters
```
Comment (dedece35) [inline tools/check_requirements/README.md:29]: please remove this line for documentation, deprecated with last upgrade

**Review by dedece35 — CHANGES_REQUESTED** [review]

File: tools/check_requirements/README.md (hunk 2 above)
Comment (d-casal) [reply tools/check_requirements/README.md]: done

**Review by dedece35 — APPROVED** [review]

**Pull request merged**